{
  Throwable cause=null;
  CertPathValidatorException tE=new CertPathValidatorException(cause);
  assertNull("getMessage() must return null.",tE.getMessage());
  assertNull("getCause() must return null",tE.getCause());
}
